srcpkgs/ngx_headers_more/INSTALL.msg

```diff
@@ -0,0 +1,4 @@
+Dynamic modules are loaded into NGINX with the `load_module` directive. Place
+the following line at the top of /etc/nginx/nginx.conf:
+
+load_module "/usr/lib/nginx/ngx_http_headers_more_filter_module.so"
```

Comment:
INSTALL messages are not the right place for documentation, this should go into a readme file if really necessary.